### PR TITLE
Added line breaks in group 2 items and formatted text in About us

### PR DESCRIPTION
**Pull request by Dhurika**

### CODE REVIEW

File: assets/css/footer.css [hunk 1]
```diff
@@ -31,10 +31,7 @@ footer > .footer-row {
   text-align: left;
 }
 
-.footer-column .sub-footer-link {
-  display: flex;
-  flex-wrap: wrap;
-}
```
Comment (kennithnichol) [inline assets/css/footer.css:37]: Why did you remove this styling?

File: index.html [hunk 2]
```diff
@@ -540,8 +540,11 @@ <h3>Enterprise</h3>
             <h2>Group 1</h2>
             <div class="sub-footer-link" role="link">
               <a href="index.html">Home</a>
+              <br>
               <a href="index.html#feature">Features</a>
+              <br>
               <a href="index.html#pricing">Pricing</a>
+              <br>
```
Comment (kennithnichol) [inline index.html:547]: This is _okay_, but it might be better to add styling for the links instead i.e. display block or flex.

File: index.html [hunk 3]
```diff
@@ -550,16 +553,19 @@ <h2>Group 1</h2>
...
               <a href="./support.html">Support</a>
             </div>
           </section>
 
           <section class="footer-column">
             <h2>About Us</h2>
             <div>
-              Keiko Corp. the greatest start up of all time Bruno the CEO of
+              Keiko Corp. The greatest start up of all time. Bruno,The CEO of
```
Comment (kennithnichol) [inline index.html:568]: 👏 Thanks for looking this over, here is some feedback!

"Keiko Corp. **the** greatest start up of all time. **Bruno, the** CEO of Keiko **Corp.,** has built the fastest growing tech start up in Silicon Valley."

Corp. is an abbreviation for Corporation, as such it's not a '.' as used in sentence punctuation.
"Bruno, the" not "Bruno, The" we would only capitalize the T at the start of a new clause, which in this case it is not!

The period after time, should be there: good eye!

File: assets/css/footer.css (hunk 1 above)
Comment (Dhurika) [reply assets/css/footer.css:37]: The items that are listed in group 1 is arranged in a horizontal manner,3 items in first row,1 item in second row. I thought it would look nice when we  arrange those items one by one.

Can I change the display to block?

File: index.html (hunk 3 above)
Comment (Dhurika) [reply index.html:568]: Thank you for the feedback .This is my first open source contribution. I will change it according to your feedback.

The comma after Bruno, Is this correct?

Comment (Dhurika) [reply index.html:568]: The socials in this footer block are also not visibl
keiko-corp
e

Comment (Dhurika) [reply index.html:568]: Can I make another pull request resolving this issues and the feedback you have given for me?
keiko-corp-updated

**Comment by kennithnichol** [comment]
You're doing great! Thanks so much for contributing and being active here!
